src/egon/data/datasets/electricity_demand_timeseries/hh_profiles.py

```diff
@@ -231,6 +232,7 @@ def __init__(self, dependencies):
                 task_id="MV-hh-electricity-load-2050",
                 python_callable=mv_grid_district_HH_electricity_load,
                 op_args=["eGon100RE", 2050],
+                op_kwargs={"drop_table": True},
```

Comment:
I am wondering what happens now when two scenarios are created. Is there a table that is then deleted and recreated for each scenario?